### PR TITLE
feat: use desktop version on Countly

### PR DESCRIPTION
**Pull request by hacdias**
Closes https://github.com/ipfs-shipyard/ipfs-desktop/issues/944.

License: MIT
Signed-off-by: Henrique Dias <hacdias@gmail.com>

### CODE REVIEW

File: src/bundles/analytics.js [hunk 1]
```diff
@@ -60,6 +60,10 @@ const createAnalyticsBundle = ({
       Countly.app_version = appVersion
       Countly.debug = debug
 
+      if (await store.selectIsIpfsDesktop()) {
```
Comment (olizilla) [inline src/bundles/analytics.js]: do we need to await here?

Comment (hacdias) [reply src/bundles/analytics.js]: Uh, perhaps not. Lemme check

**Comment by hacdias** [comment]
@olizilla removed the `await`.

File: src/bundles/analytics.js [hunk 2]
```diff
@@ -60,6 +60,10 @@ const createAnalyticsBundle = ({
       Countly.app_version = appVersion
       Countly.debug = debug
 
+      if (store.selectIsIpfsDesktop()) {
+        Countly.app_version = await store.selectDesktopVersion()
```
Comment (olizilla) [inline src/bundles/analytics.js]: await?

**Comment by hacdias** [comment]
Sorry! Fixed, @olizilla!

**Pull request merged**